src/protocols/hpr_protocol_gwmp.erl

```diff
@@ -81,15 +79,20 @@ packet_up_to_push_data(Up, GatewayTime) ->
             codr => <<"4/5">>,
             stat => 1,
             chan => 0,
-
             datr => erlang:atom_to_binary(hpr_packet_up:datarate(Up)),
             rssi => hpr_packet_up:rssi(Up),
             lsnr => hpr_packet_up:snr(Up),
             size => erlang:byte_size(hpr_packet_up:payload(Up)),
-            data => base64:encode(hpr_packet_up:payload(Up))
+            data => base64:encode(hpr_packet_up:payload(Up)),
+            meta => #{
+                gateway_id => B58,
```

Comment:
I think we need to cast this to a binary to make sure it get's encoded as a string and not a list.